packages/ListBox/src/ListBox.js

```diff
@@ -51,21 +58,30 @@ export function ListBox(props) {
     <Trigger {...propsForTrigger} />
   );
 
-  const hasOptions = Boolean(React.Children.count(children));
+  const contentProps = {
+    hasOptions,
+    onCancelFooter: footer ? footer.props.onClickCancel : null,
+    ...(isInline ? moreProps : {}),
```

Comment:
For an inline `<ListBox>` the "trigger" is just a static label, so the `moreProps` should go to the interactive `role=listbox` element.